src/relax/op/tensor/binary.cc

```diff
@@ -53,8 +53,10 @@ Optional<Expr> InferShapeBinaryBroadcast(const Call& call, DiagnosticContext dia
       } else {
         // defer the computation of output shapes to runtime
         // e.g., broadcast Tensor([m, n]), Tensor([k]) -> defer to runtime
-        return Call(ExternFunc(String("vm.binary_broadcast_shape_infer")),
-                    {call->args[0], call->args[1]}, {}, {});
+        Call call_infer(ExternFunc(String("vm.binary_broadcast_shape_infer")),
+                        {call->args[0], call->args[1]}, {}, {});
```

Comment:
maybe here worth fixing a bug in passing :). vm.binary_broadcast_shape_infer takes ShapeTuple, while here we pass two Callnode's arguments to it.

Reply:
I didn’t notice this before 🤦. Let’s fix it in the future properly given that it’s nowhere used at this moment.